docs/sources/operators-guide/tools/mimir-continuous-test.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Grafana mimir-continuous-test"
+menuTitle: "Mimir-continuous-test"
+description: "Use mimir-continuous-test to continuously run smoke tests on live Grafana Mimir clusters."
+weight: 30
+---
+
+# Grafana mimir-continuous-test
+
+Mimir-continuous-test is a standalone tool that you can use to continuously run smoke tests on live Grafana Mimir clusters.
+This tool targets Mimir developers.
+This tool aims to help identifying a class of bugs that could be difficult to spot during development with unit and integration tests.
+
+## Download mimir-continuous-test
+
+- Using Docker:
+
+```bash
+docker pull "grafana/mimir-continuous-test:latest"
+```
+
+- Using a local binary:
+
+Download the appropriate [release asset](https://github.com/grafana/mimir/releases/latest) for your operating system and architecture and make it executable.
+
+For Linux with the AMD64 architecture, execute the following command:
+
+```bash
+curl -Lo mimir-continuous-test https://github.com/grafana/mimir/releases/latest/download/mimir-continuous-test-linux-amd64
+chmod +x mimir-continuous-test
+```
+
+## Configure mimir-continuous-test
+
+Mimir-continuous-test requires the endpoints of the backend Grafana Mimir clusters and the tenant ID for writing and querying testing metrics:
+
+- Set `-tests.write-endpoint` to the base endpoint on the write path. The URL should have no trailing slash. The specific API path is appended by the tool to the URL, for example `/api/v1/push` for the remote write API.
+- Set `-tests.read-endpoint` to the base endpoint on the read path. The URL should have no trailing slash. The specific API path is appended by the tool to the URL, for example `/api/v1/query_range` for range query API.
+- Set `-tests.tenant-id` to the tenant ID to use to write and read metrics in tests.
+
+> **Note:** You can run `mimir-continuous-test -help` to list all available configuration options.
+
+## How it works
+
+Mimir-continuous-test periodically runs a suite of tests writing data to Mimir, querying it back, and checking if the query results match the expected ones.
+The tool exposes metrics that can be used to alert on test failures.
+Details about the failed tests are logged.
```

Comment:
```suggestion
The tool exposes metrics that you can use to alert on test failures, and the tool logs the details about the failed tests.
```
Unsure whether the tool logs the details or is something else does that because passive voice makes this ambiguous. Please vet this for technical accuracy.

Reply:
It's the tool logging details, so your edit is 👍